variables.tf

```
@@ -204,6 +204,24 @@ variable "storage_encrypted" {
   default     = false
 }
 
+variable "storage_type" {
+  type        = string
+  description = "One of 'standard' (magnetic), 'gp2' (general purpose SSD), or 'io1' (provisioned IOPS SSD)"
+  default     = "standard"
+}
+
+variable "iops" {
+  type        = number
+  description = "The amount of provisioned IOPS. Setting this implies a storage_type of 'io1'. Default is 0 if rds storage type is not 'io1'"
+  default     = 0
```

Can we default this to null?

```suggestion
  default     = null
```

---

I set this based on what we have in 
[terraform-aws-rds](https://github.com/cloudposse/terraform-aws-rds) module @ https://github.com/cloudposse/terraform-aws-rds/blob/558eb8f2aac9dc79b741f866bbbf8bb44d73e911/variables.tf#L80 

According to TF docs for `aws_rds_cluster`. This is an optional var, we can set this to `null`. 

On a side note though TF docs dont say any default value for this but its needed for Multi a-z cluster,  then this value ranges from min `1000` to number based on db engine, https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS